config/locales/fr.yml

```diff
@@ -0,0 +1,8 @@
+fr:
+  decidim:
+    components:
+      proposals:
+        settings:
+          global:
+            require_category: La catégorie est obligatoire
+            require_scope: Le secteur est obligatoire
```

Comment:
I think you forgot to add french translations for `form` keys

Reply:
Thanks, I verified and there is no need for the en.yml part. I used it unnecessarily when I tryed to backport the corresponding view. So I will delete the useless part!

<img width="945" alt="Capture d’écran 2024-05-21 à 16 58 31" src="https://github.com/OpenSourcePolitics/decidim-module-simple_proposal/assets/143180473/9eeb7f47-f201-4ac6-be09-2dfef7745c0f">